blocks.config.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "type": "file",
+    "id": "file-block",
+    "title": "Example File Block",
+    "description": "A basic file block",
+    "entry": "blocks/example-file-block/index.tsx",
+    "extensions": ["*"],
```

Comment:
We should use this moment to deprecate `extensions` in favor of `matches` which is more robust.

Reply:
great call!